MMUL-BASH-01/Exercises/Day-2/Exercise.006.solutions

```diff
@@ -0,0 +1,45 @@
+1) Supposing your user is "mmul" and your machine is "machine":
+
+> ssh mmul@machine
+
+2) Touch the file in order to create it
+
+touch ~/service_action
+
+3) Using our favourite editor, edit the file properly
+   (rember to save your work!)
+
+vim ~/service_action
+
+4) Touch the file and then edit the content as requested
+
+touch $(whoami)_exercise.service
+vim mmul_exercise.service
+
+5) Using the command that you already know, 'chmod'
+
+chmod 644 mmul_exercise.service
+
+6) Copy the file to the default service path
+
+sudo cp mmul_exercise.service /etc/systemd/system/
+
+7) Use the systemctl utility to make your unit available and
+   interact with it:
+
+sudo systemctl daemon-reload
+sudo systemctl start mmul_exercise.service
+
+8) Status is "Active" and the "output" of the log
+   written by our echo command:
+
```

Comment:
Please, report also the command to obtain the status

Reply:
systemctl status command has been added